app/projects/page.tsx

```diff
@@ -1,5 +1,8 @@
 import FiltersBar from '@/components/Projects/FiltersBar/FiltersBar';
 import { Metadata } from 'next';
+import ProjectCard, {
```

Comment:
I think that it would be more elegant to have `index.ts` file inside the `ProjectCard` because the import path will look more elegant, you can see what i did in `components/About/Carousel/index.ts` and how the import looks like in `pp/about/page.tsx`.